app/controllers/your-psp/get.controller.js

```diff
@@ -3,6 +3,7 @@
 const { response } = require('../../utils/response')
 const { getCredentialByExternalId, getCurrentCredential, getSwitchingCredentialIfExists, hasSwitchedProvider } = require('../../utils/credentials')
 const { getTaskList, isComplete } = require('./kyc-tasks.service')
+const enableStripeOnboardingTaskList = process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST
```

Comment:
Env variables are treated as strings so should check for exact string

```suggestion
const enableStripeOnboardingTaskList = process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST === 'true'
```